app/views/spree/admin/adjustments/_edit_form.html.haml

```diff
@@ -5,18 +5,19 @@
       = text_field :adjustment, :amount, :class => 'fullwidth'
       = f.error_message_on :amount
 
-  .four.columns
-    = f.field_container :included_tax do
-      = f.label :included_tax, t(:included_tax)
-      = f.text_field :included_tax, disabled: true, class: 'fullwidth',
-        value: number_with_precision(f.object.included_tax, precision: 2)
-      = f.error_message_on :included_tax
+  - if @adjustment.admin?
+    .four.columns
+      = f.field_container :included_tax do
+        = f.label :included_tax, t(:included_tax)
+        = f.text_field :included_tax, disabled: true, class: 'fullwidth',
+          value: number_with_precision(f.object.included_tax, precision: 2)
+        = f.error_message_on :included_tax
 
-  .omega.four.columns
-    = f.field_container :tax_rate_id do
-      = f.label :tax_rate_id, t(:tax_rate)
-      = select_tag :tax_rate_id, options_from_collection_for_select(Spree::TaxRate.all, :id, :name, @tax_rate_id), prompt: t(:remove_tax), class: 'select2 fullwidth'
-      = f.error_message_on :tax_rate_id
+    .omega.four.columns
+      = f.field_container :tax_rate_id do
+        = f.label :tax_rate_id, t(:tax_rate)
+        = select_tag :tax_rate_id, options_from_collection_for_select(Spree::TaxRate.all, :id, :name, @tax_rate_id), prompt: t(:remove_tax), class: 'select2 fullwidth'
+        = f.error_message_on :tax_rate_id
```

Comment:
I'd appreciate it if you could share a screenshot

Reply:
Admin adjustments can have a tax rate manually applied:
![Screenshot from 2021-04-20 11-30-25](https://user-images.githubusercontent.com/9029026/115381877-4fddb280-a1d4-11eb-978b-0afd0886b600.png)

Other adjustments don't use that mechanism so we don't show it (they get their tax rate a different way):
![Screenshot from 2021-04-20 11-29-13](https://user-images.githubusercontent.com/9029026/115381872-4eac8580-a1d4-11eb-98bc-cc58e45bc8dc.png)